Content.Server/Cuffs/Components/HandcuffComponent.cs

```diff
@@ -184,9 +184,15 @@ async Task<bool> IAfterInteract.AfterInteract(AfterInteractEventArgs eventArgs)
                 return true;
             }
 
-            eventArgs.User.PopupMessage(Loc.GetString("handcuff-component-start-cuffing-target-message",("targetName", eventArgs.Target)));
-            eventArgs.User.PopupMessage(target, Loc.GetString("handcuff-component-start-cuffing-by-other-message",("otherName", eventArgs.User)));
-
+            if (eventArgs.Target == eventArgs.User)
+            {
+                eventArgs.User.PopupMessage(Loc.GetString("handcuff-component-target-self"));
```

Comment:
though this is old component-code so its more passable code showing popups should avoid the `EntityUid` extension methods and should prefer the `PopupSystem` methods more or less like so:

```csharp
// this would normally be a dependency in system code, but has to be resolved for component code
EntitySystem.Get<PopupSystem>.PopupEntity(eventArgs.User, Loc.GetString("handcuff-component-target-self"), Filter.Entities(eventArgs.User);
```

similarly for the other usages of these methods, although the ones where you want to show in range (rather than just to one user) would use `Filter.Pvs`